src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -586,7 +575,8 @@ trait ChoosesMode {
           logDebug(m.toString)
       }
       val newPersonData = choosesModeData.copy(
-        personData = choosesModeData.personData.copy(currentTourMode = correctedCurrentTourMode),
+        personData = choosesModeData.personData
+          .copy(currentTourMode = if (makeCurrentTourModeNone) None else correctedCurrentTourMode),
```

Comment:
What do you mean by that?

Reply:
Once the flag set to True it will always be True, I did not see any way to set the flag back to False.
Which means this agent will always reset current tour mode - is this behavior expected?